tests/compile-fail/len_zero.rs

```diff
@@ -0,0 +1,56 @@
+#![feature(plugin)]
+#![plugin(clippy)]
+
+struct One;
+
+#[deny(len_without_is_empty)]
+impl One {
+	fn len(self: &Self) -> isize { //~ERROR
```

Comment:
Nit: keep at least the beginning bits of the error message here (and for most of the ones below too)